Source/WebCore/platform/xr/PlatformXR.h

```diff
@@ -310,8 +317,11 @@ struct FrameData {
         Vector<uint32_t> offsets;
         uint32_t fourcc;
         uint64_t modifier;
+
+        explicit operator bool() const { return !fds.isEmpty(); }
 #endif
     };
+#endif // OS(ANDROID)
```

Comment:
I find this a bit confusing when there's an else. I think I would omit the comment.